docs/buildExternalDocs.bash

```diff
@@ -12,7 +12,7 @@ cp -a ../merlin-server/constraints-dsl-compiler/build/docs/. ./source/constraint
 rm -f ./source/constraints-edsl-api/.nojekyll
 
 mkdir -p ./source/scheduling-edsl-api
-cp -a ../scheduler-server/scheduling-dsl-compiler/build/docs/. ./source/scheduling-edsl-api
+cp -a ../scheduler-worker/scheduling-dsl-compiler/build/docs/. ./source/scheduling-edsl-api
```

Comment:
`docs-pages.yaml` and `docs-pr.yaml` need this same change in the line `"scheduler-server/scheduling-dsl-compiler/src/**"`